frontends/election-manager/src/lib/backends/admin_backend.ts

```diff
@@ -89,6 +92,57 @@ export class ElectionManagerStoreAdminBackend extends ElectionManagerStoreStorag
     return parsedElectionDefinition;
   }
 
+  async addCastVoteRecordFile(
```

Comment:
Where are we determining which store/backend we're using?

Reply:
https://github.com/votingworks/vxsuite/blob/4478ce90d99388af32d63eb3c654e0fd6a9b4d79/frontends/election-manager/src/index.tsx#L16

Reply:
gotcha, I missed that the call to super here is how we are also storing it in the old location